app/views/tweets/tweets.erb

```diff
@@ -0,0 +1,9 @@
+Welcome, <%= current_user.username %>
```

Comment:
Here's an example of where you call the `current_user` a second time causing a second db query, it would be doubly useful to avoid that in this instance since we try to avoid making db calls from the views.